VultisigApp/VultisigApp/Views/Keygen/KeyGenSummaryView.swift

```diff
@@ -69,6 +77,24 @@ struct KeyGenSummaryView: View {
         .font(.body20MontserratSemiBold)
     }
     
+    var numberOfDevicesForReshareTitle: some View {
+        VStack(spacing: 16) {
+            Text(NSLocalizedString("newVaultSetup", comment: ""))
+                .font(.body14MenloBold)
+            
+            Group {
```

Comment:
<img width="1245" alt="image" src="https://github.com/user-attachments/assets/f240d03e-b8dd-429a-abb0-cfaf690a6999">
I think the logic is a bit off for this , in this case , I am trying to add my pixel 8 to the group, which should say 
new vault setup , 2/3

Pixel 8 is the device get added , it should be green color , not red , red means it will be removed